wizeline/de-academy
# Updates setup script to work

## Overview
Updates the setup script to ensure it all works fine and makes it a bit more user-friendly.

## How to test it?
1. Run `./setup.sh`
2. If you have a previous version of `gcloud` installed, the script should detect it and warn you about it. It will skip some steps but still configure the DE Academy project.
3. If you don't have the `gcloud` SDK previously installed, you should be able to just follow the on-screen instructions, relax and wait for the successful installation message at the end.

## Notes
This has been tested in `Ubuntu 16.04.5 LTS (Xenial)` and `Mac OS X 10.9.5 (Mavericks)`
